### DISCUSSION TITLE
lang setting

### DISCUSSION THREAD

**Discussion by corean** [opcodesio/log-viewer#232] [Ideas]
```php
# config/app.php 
'locale' => 'ko'
```
to set the locale of the log page.

![screenshot 2023-06-19 오후 8 29 06](https://github.com/opcodesio/log-viewer/assets/225376/8a0f2454-3814-4b89-92d7-5cded0bd07d8)

**Comment by arukompas** [comment]
Hey, Log Viewer currently does not support multiple languages, but feel free to submit a PR! :)
